test/fixtures.yml

```diff
@@ -219,6 +219,11 @@ credorax:
   merchant_id: 'merchant_id'
   cipher_key: 'cipher_key'
 
+# Working credentials, no need to replace
```

Comment:
Okay, last thing: I doubt these are working credentials, since they're the default, but if they are, never mind and we're good to go. (I suspect for them we just want to axe out the comment, not put in working creds.)

Reply:
Fixed. Thanks @bpollack for all these catches.

Reply:
You're welcome! You're good to ship now, just to reiterate.

Reply:
Waiting for certification test to be approved so they can give us the actual live url.